script/M1_deploy.config.json

```diff
@@ -37,13 +37,15 @@
      "eigenPod":
           {
                "PARTIAL_WITHDRAWAL_FRAUD_PROOF_PERIOD_BLOCKS": 50400,
-               "REQUIRED_BALANCE_WEI": "31000000000000000000"
-          },
-     "eigenPodManager":
-          {    
-               "max_pods": 0,
-               "init_paused_status": 30
+               "REQUIRED_BALANCE_WEI": "31000000000000000000",
+               "MAX_VALIDATOR_BALANCE_GWEI": "32000000000000000000",
+               "EFFECTIVE_RESTAKED_BALANCE_OFFSET": "750000000"
```

Comment:
seems really tiny?

Reply:
no here MAX_VALIDATOR_BALANCE_GWEI is in WEI which is wrong, fixed again.